Assets/MixedRealityToolkit-SDK/StandardAssets/Controllers/Hands/LICENSE.MD

```diff
@@ -0,0 +1,98 @@
+
+1.2 For the sake of clarification, when you use the Oculus SDK in or with Developer Content, you retain all rights to your Developer Content, and you have no obligations to share or license Developer Content (including your source and object code) to Oculus or any third parties; provided, however, Oculus retains all rights to the Oculus SDK and the headers, libraries and APIs to the Platform Services and other tools made available by Oculus, including those that may be incorporated into your Developer Content.
+
+1.3 You agree that as a condition of this License you will design and distribute your Developer Content to ensure that your Developer Content and any software required to use your Developer Content does not, and you will not, alter or interfere with the normal operation, behavior or functionality of the Oculus hardware or software or Oculus Approved Products, including: (i) the behavior of the "Oculus button" and "XBox button" implemented by the Oculus system software; (ii) any on-screen messages or information; (iii) the behavior of the proximity sensor in the Oculus hardware implemented by the Oculus system software; (iv) Oculus hardware or software security features; (v) end user's settings; or (vi) the Oculus Flash Screen Warnings. You also agree not to commit any act intended to interfere with the normal operation of the Oculus hardware or software or Oculus Approved Products, or provide software to Oculus users or developers that would induce breach of any Oculus agreements or that contains malware, viruses, hacks, bots, Trojan horses, or other malicious code.
+
+1.4 You may not use the Oculus SDK for any purpose not expressly permitted by this License. You may not:
+
+* decompile
+* reverse engineer
+* disassemble
+* attempt to derive the source code of the Oculus SDK or any part of the Oculus SDK, or any other software or firmware provided to you by Oculus
+(except as and only to the extent any foregoing restriction is prohibited by applicable law).
+
+## REDISTRIBUTION
+
+2. Subject to the terms and conditions of this License, your license to redistribute and sublicense the Oculus SDK is also expressly made subject to the following conditions:
+
+2.1 You may sublicense and redistribute the source, binary, or object code of the Oculus SDK in whole for no charge or as part of a for-charge piece of Developer Content; provided, however, you may only license, sublicense or redistribute the source, binary or object code of the Oculus SDK in its entirety. The Oculus SDK (including, but not limited to LibOVR and VRLib), and any Developer Content that includes any portion of the Oculus SDK, may only be used with Oculus Approved Products and may not be used, licensed, or sublicensed to interface with software or hardware or other commercial headsets, mobile tablets or phones that are not authorized and approved by Oculus;
```

Comment:
Are we following this license correctly? Particularly (emphasis mine):

>you may only license, sublicense or redistribute the source, binary or object code of the Oculus SDK in its **entirety**

and

>may only be used with Oculus Approved Products and may not be used, licensed, or sublicensed to interface with software or hardware or **other commercial headsets**, mobile tablets or phones that are not authorized and approved by Oculus

Reply:
I believe so, as we took the entire controller bundle for both the hands and the touch controllers.
Each are downloaded separately

Reply:
Sure, I'll remove them and add links to where they can download them